### ISSUE TITLE
save in browser dark/light theme state

### ISSUE THREAD

**Issue by muravikov** [bitrix24/b24restdocs#3]
in first time open documentation as default light theme, okay. 
but if i change theme to dark, this state not saved :(

**Comment by serg-vostrikov-bitrix** [comment]
yes, we know. unfortunately, we depend on the documentation platform we use. when they add this feature, we'll also use it.
